doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,258 @@
+| vim_name | String | Yes | vim name |
+| vim_id | String | No | a name VIM Resource pool naming rule |
+| vendor | String | Yes | VIM Provider information, e.g. ZTE,Huawei,Ericsson,NOKIA|
+| version | String | Yes | e.g NFV-D-HDBNJ-02A-3503-G-02-M-SRV-01 |
+| ip_version | String | Yes | Ipaddress type: IPV6 or IPV4 |
+| controller_nodes | List | Yes | List of controllers designed in VIM deployment |
+| compute_nodes | List | Yes | List of compute nodes designed in VIM deployment |
+
+<p align="center"><b>Table 5-3-4:</b> Software configuration.</p>
+
+
+### 5.3.5 Network informations
+List of All designed NIC definitions which are referenced by various roles of node, control/compute/network/storage node.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| network_plane_type | String | Yes | distinguish network type: e.g MANAGEMENT,STORAGEDATA |
+| usage | String | Yes | used for compute/control: value is one of [manage,storage,service,PXE,iLo], for distribute storage  value if one of [storage_front,storage_backend] |
+| network_plane_infos | List | Yes | vlan_id,ip address,sub_network(CIDR),gateway|
+
+<p align="center"><b>Table 5-3-5-1:</b> Network information.</p>
+
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| vlan_id | String | Yes | vlan id for current network |
+| ip_address | String | Yes | ip address |
+| sub_network | String | Yes | subnet in CIDR format. e.g: 2409:8086:8412:100::/64|
+| gateway | String | No | gateway|
+
+<p align="center"><b>Table 5-3-5-2:</b> Network plane information.</p>
+
+
+
+### 5.3.6 Controller informations
+List of controller nodes that designed for current VIM deployment.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| device_name | String | Yes | must be the device_name defined in server_information(###5.3.3), which would be set on Server from BMC|
+| node_name | String | Yes | node name for the controller to deploy, e.g: controller01 |
+| az_name | String | Yes | availability zone name which this node belongs to |
+| ha_name | List | Yes | HA name which this node assigned to, could be more than one HA it belongs.|
```

Comment:
What does "ha" acronym represent? If host aggregate then controller nodes are not made part of host aggregates. BTW neither are they made part of an Availability Zone.

Reply:
OK, Thanks for your comments. I'll correct it.

Reply:
OK, Thanks for your comments. I'll correct it.